ui/src/PaperSummary.tsx

```diff
@@ -101,6 +101,7 @@ export class PaperSummary extends React.PureComponent<
           const hasMetrics = paper.citationVelocity !== 0 || paper.influentialCitationCount !== 0;
           const truncatedAbstract = paper.abstract ? truncateText(paper.abstract, 300) : null;
           const inLibrary = userLibrary ? userLibrary.paperIds.includes(this.props.paperId) : false;
+          const summaryUrl = `https://www.semanticscholar.org/paper/{paper.Id}`
```

Comment:
Should this be `id` with a lower case `i`?
```suggestion
          const summaryUrl = `https://www.semanticscholar.org/paper/{paper.id}`
```